packages/core/src/components/Dropdown/Dropdown.tsx

```diff
@@ -432,7 +434,7 @@ const Dropdown: VibeComponent<DropdownComponentProps, HTMLElement> & {
         placeholder={placeholder}
         isDisabled={disabled}
         isClearable={!readOnly && clearable}
-        isSearchable={!readOnly && searchable}
+        isSearchable={!readOnly}
```

Comment:
why is it removed? intentional?

Reply:
Yep, I stopped sending the searchable prop to react-select because when it's set, the library creates a dummy input that is missing the proper aria labels. Instead, I handle the behavior directly in the Input component to make sure the accessibility labels are correct.